docs/source/i524/lectures-theory.csv

```diff
@@ -11,11 +11,11 @@ Introduction ,Course Introduction,`Slides <https://drive.google.com/file/d/0Bx_s
       ,Introduction,`Video <https://www.youtube.com/watch?v=VWK7QdThOPQ>`_,0:13:59
       ,Introduction - Real World Big Data,`Video <https://www.youtube.com/watch?v=hjrFaXMU1dg&list=PLLO4AVszo1SNIGlqjEBaQjLvc3lgpeI3M&index=2>`_,0:15:28
       ,Introduction - Basic Trends and Jobs,`Video <https://www.youtube.com/watch?v=u5Y0aKbzync&index=3&list=PLLO4AVszo1SNIGlqjEBaQjLvc3lgpeI3M>`_,0:10:57
-Acess Patterns, Data Access Patterns and Introduction to using HPC-ABDS,`Slides <https://iu.box.com/s/uozssvsy0p9fhh0uv8gldwoves2bl0vv>`_,
+Acess Patterns, Data Access Patterns and Introduction to using HPC-ABDS,`Slides <https://iu.app.box.com/files/0/f/16611433111/1/f_122710787684>`_,
```

Comment:
This link requires me to log into box.
Please change this to one that does not.
You can test by opening in an incognito window.